src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngine.java

```diff
@@ -477,36 +485,88 @@ protected static GATKSVVCFConstants.StructuralVariantAnnotationType getSVType(fi
      * Add protein-coding annotations for any transcripts overlapping the variant to the variant consequence dictionary
      * @param variantInterval - SimpleInterval representing structural variant
      * @param svType - SV type
+     * @param isComplex - boolean: true if SV type is CPX, false otherwise
      * @param variantConsequenceDict - running map of consequence -> feature name for variant to update
      */
     @VisibleForTesting
     protected void annotateGeneOverlaps(final SimpleInterval variantInterval,
                                                final GATKSVVCFConstants.StructuralVariantAnnotationType svType,
+                                               final boolean isComplex,
                                                final Map<String, Set<String>> variantConsequenceDict) {
         final Iterator<SVIntervalTree.Entry<GencodeGtfTranscriptFeature>> gtfTranscriptsForVariant =
                 gtfIntervalTrees.getTranscriptIntervalTree().overlappers(
                         SVUtils.locatableToSVInterval(variantInterval, sequenceDictionary)
                 );
         for (Iterator<SVIntervalTree.Entry<GencodeGtfTranscriptFeature>> it = gtfTranscriptsForVariant; it.hasNext(); ) {
             SVIntervalTree.Entry<GencodeGtfTranscriptFeature> transcriptEntry = it.next();
-            annotateTranscript(variantInterval, svType, transcriptEntry.getValue(), variantConsequenceDict);
+            annotateTranscript(variantInterval, svType, isComplex, transcriptEntry.getValue(), variantConsequenceDict);
+        }
+    }
+
+    /**
+     * Get section of one interval (primaryInterval) that is not overlapped by the other (secondaryInterval)
+     * @param primaryInterval - SimpleInterval
+     * @param secondaryInterval - SimpleInterval overlapping (but not fully containing) primaryInterval
+     * @return - SimpleInterval representing the portion of primaryInterval not overlapped by secondaryInterval
+     */
+    @VisibleForTesting
+    protected static SimpleInterval getNonOverlappingInterval(final SimpleInterval primaryInterval,
```

Comment:
I started doing this originally but then realized that it's not super generalizable because I made a lot of assumptions about the intervals. I could still add it but would need to enforce the secondary interval overlapping but not containing the primary interval